docs/cimpl6ToolingReference.md

```diff
@@ -469,7 +468,7 @@ The content of the _/out_ directory depends on which exporters were selected to
 * _cimcore_ - this directory is only used in the process of building the `modeldoc` export, and may not appear in future releases.
 * [_data-dictionary_](#data-dictionary-export) - this directory contains an MS-Excel spreadsheet containing a list of model `Element`s and value sets
 * [_fhir_](#fhir-export) - this directory contains all the definitions and assets necessary to produce the IG
-* [_json-schema_](#json-schema-export) - this directory contains schemas for the (**TO DO -- need to define the JSON schema export**)
+* [_json-schema_](#json-schema-export) - this directory contains schemas for the (**TO DO -- define the JSON schema export**)
 * [_modeldoc_](#model-documentation-export) - this directory contains files that present the model with a look and feel similar to [Javadoc](https://www.oreilly.com/library/view/learn-to-program/9781680500523/f_0126.html)
 
```

Comment:
Can you add documentation for the Dylan's new graphical view?

Reply:
OK, will remove all references to json schema export, and start working on the graphical view.

Reply:
@markkramerus   Does this mean any references to entryTypeURL should also be removed?